src/biogeophys/WaterStateType.F90

```diff
@@ -586,6 +587,7 @@ subroutine Restart(this, bounds, ncid, flag, &
          dim1name='column', dim2name='levtot', switchdim=.true., &
          long_name=this%info%lname('ice lens'), &
          units='kg/m2', &
+         scale_by_thickness=.true., &
```

Comment:
Right off the bat I can tell that also `SMP` and `HK` need `scale_by_thickness = .true.` because they have `units = mm` and `units = mm/s`.

Then there's a long list of C & N _vr variables with their units unspecified in the finidat file. I will investigate those.